src/js/components/PageHeader.js

```diff
@@ -121,7 +131,8 @@ PageHeader.propTypes = {
   iconClassName: React.PropTypes.oneOfType([
     React.PropTypes.string,
     React.PropTypes.object,
-  ])
+  ]),
+  subTitleClassName: React.PropTypes.string,
```

Comment:
ugh, it was introduced in this PR (https://github.com/dcos/dcos-ui/commit/95154b3523f3f7547def875a60846fdff58a3b20). Yes definitely agreed then.